pkg/health/health.go

```diff
@@ -61,9 +72,14 @@ func (h ConsulHealthChecker) WatchNodeService(nodename string, serviceID string,
 			})
 			if err != nil {
 				errCh <- err
+			}
+			kvCheck, err := h.consulStore.GetHealth(nodename, serviceID)
```

Comment:
Is there an error if the health is not present for that node? It needs to be the case that a service:node is healthy when either health checks are present, not necessarily both.